### PR TITLE
Functional tests - Add new test to check 'Payment block in view order page'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test to check 'Payment block in view order page'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26615)

### CODE REVIEW

File: tests/UI/pages/BO/orders/view.js [hunk 1]
```diff
@@ -864,6 +890,95 @@ class Order extends BOBasePage {
...
+    await this.setValue(page, this.paymentMethodInput, paymentData.paymentMethod);
+    await this.setValue(page, this.transactionIDInput, paymentData.transactionID);
+    await this.setValue(page, this.paymentAmountInput, paymentData.amount);
+    if (paymentData.currency !== '€') {
+      await this.selectByVisibleText(page, this.paymentCurrencySelect, paymentData.currency);
+    }
+    if (invoice !== '') {
+      await this.selectByVisibleText(page, this.paymentInvoiceSelect, invoice);
+    }
+    await this.clickAndWaitForNavigation(page, this.paymentAddButton);
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/orders/view.js]:
```suggestion
    await this.setValue(page, this.paymentAmountInput, paymentData.amount);
    if (paymentData.currency !== '€') {
      await this.selectByVisibleText(page, this.paymentCurrencySelect, paymentData.currency);
    }

    if (invoice !== '') {
      await this.selectByVisibleText(page, this.paymentInvoiceSelect, invoice);
    }

    await this.clickAndWaitForNavigation(page, this.paymentAddButton);
```

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js [hunk 2]
```diff
@@ -0,0 +1,671 @@
...
+        .to.equal(`Warning €${totalOrder} paid instead of €57.74`);
+    });
+
+    it('should add payment when amount is equal to the rest to the new invoice', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'testAmountSupTotal', baseContext);
+
+      let invoice = '';
+      if (invoiceID >= 10) {
+        invoice = `#IN0000${invoiceID + 1}`;
+      } else invoice = `#IN00000${invoiceID + 1}`;
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js]:
```suggestion
      const invoice = `#IN0000${invoiceID >= 10 ? '0' : ''}${invoiceID + 1}`;
```

**Review by Progi1984 — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/08_paymentBlock.js]: Done thx :+1:

File: tests/UI/pages/BO/orders/view.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view.js]: Done thx :+1:

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**